pkg/bootstrap/platform/azure.go

```diff
@@ -104,30 +104,32 @@ func (e *azureEnv) parseMetadata(metadata string) {
 	}
 }
 
+const emptyJSONResponse = "{}"
+
 // Generic Azure metadata GET request helper for the response body
 // Uses the default timeout for the HTTP get request
 func metadataRequest(query string) string {
 	client := http.Client{Timeout: defaultTimeout}
 	req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("%s?%s", AzureInstanceURL, query), nil)
 	if err != nil {
-		log.Warnf("Failed to create HTTP request: %v", err)
```

Comment:
Wouldn't a lack of access to the metadata service break locality loadbalancing and some telemetry? Warn feels like the correct log level

Reply:
I thinks we did same thing to AWS and GCP?

Reply:
Oh if so then disregard. I'm all for consistency